### PR TITLE
Create dir 2.x and 3.x

### PR DESCRIPTION
**Pull request by QXG2**
At a minimum please provide the following:

### Description of the Change

Create folder 2.x in tutorial. Move gsql101 and gsql102 into it. And copy 2.x to another folder 3.x. Prepare to make updates in 3.x.

### CODE REVIEW

File: demos/guru_scripts/docker/tutorial/3.x/gsql101/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+Documents
+==============
+https://docs.tigergraph.com/intro/gsql-101
+
+
+Support
+===============
+If you like the tutorial and want to explore more, join the GSQL developer community at 
+
+https://groups.google.com/a/opengsql.org/forum/?hl=en#!forum/gsql-users
```
Comment (mingxiw) [inline demos/guru_scripts/docker/tutorial/3.x/gsql101/README.md:24]: this google group has expired. We moved to this
https://community.tigergraph.com/

So, please check all such links, and update them to the above community link.

File: demos/guru_scripts/docker/tutorial/3.x/gsql101/gsql101.gsql [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+   LOAD file1 TO VERTEX person VALUES ($"name", $"name", $"age", $"gender", $"state") USING header="true", separator=",";
+   LOAD file2 TO EDGE friendship VALUES ($0, $1, $2) USING header="true", separator=",";
+}
+
+#run the loading job
+RUN LOADING JOB load_social
+
+#built-in queries
+Use Graph Social
```
Comment (mingxiw) [inline demos/guru_scripts/docker/tutorial/3.x/gsql101/gsql101.gsql:23]: Please read GSQL 101, 3.0 doc
https://docs.tigergraph.com/start/gsql-101/built-in-select-queries

We have removed all SELECT count(*) queries. We use REST call instead. Please copy those restcall curl commands here, See the above link.

File: demos/guru_scripts/docker/tutorial/3.x/gsql101/gsql101.gsql [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+SELECT * FROM person WHERE primary_id=="Tom"
+SELECT name FROM person WHERE state=="ca"
+SELECT name, age FROM person WHERE age > 30
+
+SELECT * FROM person-(friendship)->person WHERE from_id =="Tom"
+SELECT * FROM person-(ANY)->ANY WHERE from_id =="Tom"
+
+#create a stored procedure.
+CREATE QUERY hello(VERTEX<person> p) FOR GRAPH social{
```
Comment (mingxiw) [inline demos/guru_scripts/docker/tutorial/3.x/gsql101/gsql101.gsql:35]: we still keep CREATE QUERY hello ()... and below.

File: demos/guru_scripts/docker/tutorial/3.x/gsql102/README.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+Documents
+==============
+https://docs.tigergraph.com/intro/gsql-102
+
+Support
+===============
+If you like the tutorial and want to explore more, join the GSQL developer community at 
+
+https://groups.google.com/a/opengsql.org/forum/?hl=en#!forum/gsql-users
```
Comment (mingxiw) [inline demos/guru_scripts/docker/tutorial/3.x/gsql102/README.md:55]: use community.tigergraph.com, this link expired.

File: demos/guru_scripts/docker/tutorial/3.x/gsql102/queries/multiple-hop1.gsql [hunk 5]
```diff
@@ -0,0 +1,14 @@
+USE GRAPH ldbc_snb
+SET syntax_version="v2"
+
+INTERPRET QUERY () FOR GRAPH ldbc_snb {
+
+  TagClass1 = {TagClass.*};
+
+  TagClass2 =  SELECT t
```
Comment (mingxiw) [inline demos/guru_scripts/docker/tutorial/3.x/gsql102/queries/multiple-hop1.gsql:8]: please read gsql 102 of 3.0 doc, this query has changed. We can start from TagClass directly.

File: demos/guru_scripts/docker/tutorial/3.x/gsql102/queries/multiple-hop2.gsql [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+INTERPRET QUERY () FOR GRAPH ldbc_snb {
+
+  SumAccum<String> @continentName;
+
+  msg = {Comment.*, Post.*};
+
+  accMsgContinent =
+    SELECT s
+    FROM msg:s-((Post_IS_LOCATED_IN_Country>|Comment_IS_LOCATED_IN_Country>).Country_IS_PART_OF_Continent>)-Continent:t
```
Comment (mingxiw) [inline demos/guru_scripts/docker/tutorial/3.x/gsql102/queries/multiple-hop2.gsql:12]: please check https://docs.tigergraph.com/start/gsql-102/get-set

3.0 doc has changed these queries.

File: demos/guru_scripts/docker/tutorial/3.x/gsql102/queries/multiple-hop3.gsql [hunk 7]
```diff
@@ -0,0 +1,14 @@
+USE GRAPH ldbc_snb
+SET syntax_version="v2"
+
+INTERPRET QUERY () FOR GRAPH ldbc_snb {
+  SumAccum<int> @likesCnt;
+  PersonAll = {Person.*};
+
+  FavoriteAuthors =  SELECT t
+               FROM PersonAll:s-((Person_LIKES_Comment>|Person_LIKES_Post>)) -:msg - ((Comment_HAS_CREATOR_Person>|Post_HAS_CREATOR_Person>))-Person:t
```
Comment (mingxiw) [inline demos/guru_scripts/docker/tutorial/3.x/gsql102/queries/multiple-hop3.gsql:9]: check 3.0 doc.

File: demos/guru_scripts/docker/tutorial/3.x/gsql102/queries/one-hop-star1.gsql [hunk 8]
```diff
@@ -0,0 +1,13 @@
+USE GRAPH ldbc_snb
+SET syntax_version="v2"
+
+INTERPRET QUERY () FOR GRAPH ldbc_snb {
+
+  TagClass1 = {TagClass.*};
+
```
Comment (mingxiw) [inline demos/guru_scripts/docker/tutorial/3.x/gsql102/queries/one-hop-star1.gsql:7]: check 3.0 doc.

File: demos/guru_scripts/docker/tutorial/3.x/gsql102/queries/one-hop-star2.gsql [hunk 9]
```diff
@@ -0,0 +1,13 @@
+USE GRAPH ldbc_snb
+SET syntax_version="v2"
+
+INTERPRET QUERY () FOR GRAPH ldbc_snb {
+
+  TagClass1 = {TagClass.*};
+
+  TagClass2 =  SELECT t
+               FROM TagClass1:s - (TagClass_IS_SUBCLASS_OF_TagClass>*1)-TagClass:t
+               WHERE s.name == "TennisPlayer";
```
Comment (mingxiw) [inline demos/guru_scripts/docker/tutorial/3.x/gsql102/queries/one-hop-star2.gsql:10]: 3.0 doc.

File: demos/guru_scripts/docker/tutorial/3.x/gsql102/queries/one-hop-star3.gsql [hunk 10]
```diff
@@ -0,0 +1,13 @@
+USE GRAPH ldbc_snb
+SET syntax_version="v2"
+
+INTERPRET QUERY () FOR GRAPH ldbc_snb {
+
+  TagClass1 = {TagClass.*};
+
+  TagClass2 =  SELECT t
+               FROM TagClass1:s - (TagClass_IS_SUBCLASS_OF_TagClass>*1..2)-TagClass:t
```
Comment (mingxiw) [inline demos/guru_scripts/docker/tutorial/3.x/gsql102/queries/one-hop-star3.gsql:9]: 3.0 doc.

File: demos/guru_scripts/docker/tutorial/3.x/gsql102/queries/one-hop-star4.gsql [hunk 11]
```diff
@@ -0,0 +1,13 @@
+USE GRAPH ldbc_snb
+SET syntax_version="v2"
+
+INTERPRET QUERY () FOR GRAPH ldbc_snb {
+
+  TagClass1 = {TagClass.*};
+
+  TagClass2 =  SELECT t
+               FROM TagClass1:s - (TagClass_IS_SUBCLASS_OF_TagClass>*..2)-TagClass:t
```
Comment (mingxiw) [inline demos/guru_scripts/docker/tutorial/3.x/gsql102/queries/one-hop-star4.gsql:9]: 3.0 doc.

**Comment by QXG2** [comment]
> many things have been changed in 3.0 doc. Please read carefully and made changes here.
> 
> This is still copy 2.x queries.

Thanks for comment! This is a wrong pr. I haven't committed other modifications yet.

**Pull request closed without merging**